sphinx/tutorial/rose/metadata.rst

```diff
@@ -66,7 +66,8 @@ The Metadata Format
 
    [env=WORLD]
    description=The name of the world to say hello to.
-   values=Mercury, Venus, Earth, Mars, Jupiter, Saturn, Uranus, Neptune
+   values=Mercury, Venus, Earth, Mars, Jupiter,
+         =Saturn, Uranus, Neptune
```

Comment:
To stop the text getting cut off by the box end, I am sure, but this will also be a nice reminder on multi-line value usage, which is mentioned for the Rose config format but then not used in any of the examples.